prebuild_binaries/native/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:10-jessie
```

Comment:
I am not sure what "native" means in this directory name of this file. The term "native" gets overloaded a lot, just avoid it.

Reply:
Switched to "linux" -- not sure of a short way to refer to non-alpine-linux.